docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -256,3 +256,69 @@ const theme = createTheme({
   },
 });
 ```
+
+## Stop passing `utils` and the date/time object to some translation keys
+
+If you are customizing the translation of those keys, you no longer need to provide `utils` and the date/time object to it, but only the formatted date/time value as a string.
```

Comment:
Grammarly suggestion. 😉 
```suggestion
If you are customizing the translation of those keys, you no longer need to provide `utils` and the date object, but only the formatted date value as a string.
```

Reply:
Oh, sorry for the confusion.
The removal of "to it" was suggested by Grammarly, but the `/time` removal was from me. 🙈 
I think that we should use Date and Time Pickers if we want to refer to the components,
In this context it feels like `date/time` refers to the date object mainly. 🤷